PowerShellTools/d0kijkcg.tmp_proj

```diff
@@ -0,0 +1,732 @@
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
remove this tmp file from commit.
[These being around seem to be new... do we want to add .tmp_proj to this project's .gitignore?]